tokio/src/net/tcp/stream.rs

```diff
@@ -883,10 +864,10 @@ impl TcpStream {
     /// this comes at the cost of a heap allocation.
     ///
     /// **Note:** Dropping the write half will shut down the write half of the TCP
-    /// stream. This is equivalent to calling [`shutdown(Write)`] on the `TcpStream`.
+    /// stream. This is equivalent to calling [`shutdown_std(Write)`] on the `TcpStream`.
     ///
     /// [`split`]: TcpStream::split()
-    /// [`shutdown(Write)`]: fn@crate::net::TcpStream::shutdown
+    /// [`shutdown_std(Write)`]: fn@crate::net::TcpStream::shutdown_std
```

Comment:
This needs to be changed.

Reply:
@Darksonn I changed those to `shutdown_std(Write)`. Are you referring to before those changes?

Reply:
There was a docs error in CI before I made said changes, but seems to pass now.